README.md

```diff
@@ -32,10 +32,18 @@ At this step you will need to enter your Valispace url (e.g. https://demo.valisp
 
 Then use the Valispace API like this:
 
-Get a dict of all Valis:
+Get a dict of an entire data type:
 
 ```
-all_valis = valispace.all_valis()
+all_valis = valispace.all_data(type='vali')
+```
+
+The type field can be a 'component', 'vali', 'textvali' or 'tag'
+
+Post new data:
+
+```
+all_valis = valispace.post_data(type='vali')
```

Comment:
- The return of this function is probably not all_valis, so maybe give it a different name to avoid confusion?
- There should be an data field next to the type field, otherwise this will return an error as defined in the function
- Add an example of valid input data?